saltgui/static/scripts/Api.js

```diff
@@ -41,10 +41,12 @@ export class API {
   }
 
   static _cleanStorage () {
-    // clear local storage except key 'eauth'
+    // clear local storage except key 'eauth' and 'runtype'
     const eauth = Utils.getStorageItem("local", "eauth");
+    const runtype = Utils.getStorageItem("local", "runtype");
     Utils.clearStorage("local");
     Utils.setStorageItem("local", "eauth", eauth);
+    Utils.setStorageItem("local", "runtype", runtype);
 
```

Comment:
actually, I completely forgot that we erase most local storage, but this handles that part nicely